packages/commonwealth/client/styles/components/component_kit/new_designs/cw_tooltip.scss

```diff
@@ -0,0 +1,84 @@
+@import '../../../shared';
+
+@mixin tip ($top, $right, $bottom, $left) {
+  position: absolute;
+  transform: rotate(45deg);
+  width: 8px;
+  height: 8px;
+
+  background: $neutral-900;
+  
+  flex: none;
+  order: 1;
+  flex-grow: 0;
+  z-index: 1;
+
+  top: $top;
+  right: $right;
+  bottom: $bottom;
+  left: $left;
+}
+
+.Tooltip {
```

Comment:
We already have class `Tooltip` in the code so when you have two Tooltips in the page (the old one and the new one, the styles are clashed). My suggestion is that you should change `Tooltip` class into `OldTooltip` in  old `cw_popover/cw_tooltip.scss`

Reply:
Nice catch. Thanks for this!

Reply:
Addressed